.github/workflows/ros-build-test.yml

```diff
@@ -46,6 +46,16 @@ jobs:
         git clone --recurse-submodules https://github.com/leggedrobotics/pinocchio.git src/pinocchio
         git clone --recurse-submodules https://github.com/leggedrobotics/hpp-fcl.git src/hpp-fcl
         git clone https://github.com/leggedrobotics/ocs2_robotic_assets.git src/ocs2_robotic_assets
+        
+    - name: Install RaiSim
+      run: |
+        git clone --depth 1 https://github.com/raisimTech/raisimLib.git -b v1.1.01 src/raisim_tech
+        cd src/raisim_tech
+        mkdir build
+        cd build
+        cmake .. -DRAISIM_EXAMPLE=ON -DRAISIM_PY=ON -DPYTHON_EXECUTABLE=$(python3 -c "import sys; print(sys.executable)")
```

Comment:
If it is possible, let's remove the option for RAISIM_EXAMPLE and RAISIM_PY. We are using raisim python interface.